src/core/render-core.js

```diff
@@ -150,6 +165,10 @@ async function render(_opts = {}) {
 
       data = await page.screenshot(screenshotOpts);
     }
+
+    // Write data to file
+    logger.info('[renderCore] write to file, jobId = ', opts.jobId);
```

Comment:
@tungsheng : Do we have a way to direct logging to a file?

Reply:
@visthakk I think so, should be able to save it to a text file in HDFS or other options. When do you think we will need to implement this logging feature?

Reply:
Log rotation added in deploy script.